src/test/java/com/google/collegeplanner/TestPlanner.java

```diff
@@ -0,0 +1,43 @@
+package com.google.collegeplanner;
```

Comment:
sorry, I forgot what the other files were, but test files should be `PlannerTest.java` (not `TestPlanner.java`). We should change the others if they don't conform.